src/pybind/mgr/dashboard/frontend/src/app/shared/services/prometheus-alert.service.ts

```diff
@@ -57,9 +58,15 @@ export class PrometheusAlertService {
   }
 
   private handleAlerts(alerts: AlertmanagerAlert[]) {
+    this.silencedAlerts = 0;
```

Comment:
Wouldn't it work even without this added line?

Reply:
Since this method is called in a loop, the variable needs to be reinitialised to 0 after every call.